shell/platform/tizen/flutter_tizen_ecore.cc

```diff
@@ -52,3 +52,23 @@ FlutterDesktopViewRef FlutterDesktopViewCreateFromNewWindow(
 
   return HandleForView(view.release());
 }
+
+FlutterDesktopViewRef FlutterDesktopViewCreateFromElmParent(
+    const FlutterDesktopViewProperties& view_properties,
+    FlutterDesktopEngineRef engine,
+    void* parent) {
+  // Not supported.
```

Comment:
These are public APIs, so it would be nice to provide proper logs.
and I think that level should be warn.

Reply:
I added a FIXME comment and FT_UNIMPLEMENTED() . Please review again if this is appropriate